shared/src/commonMain/sqldelight/local/TrackingTask.sq

```diff
@@ -0,0 +1,68 @@
+import io.customer.shared.sdk.meta.IdentityType;
+import io.customer.shared.tracking.api.model.TrackingErrorReason;
+import io.customer.shared.tracking.constant.QueueTaskStatus;
+import kotlinx.datetime.Instant;
+
+-- TABLE CREATIONS
+-- Any updates in the table should only be done with proper migrations
+
+CREATE TABLE IF NOT EXISTS trackingTask (
```

Comment:
This table seems a bit long compared to the amount of data the background queue currently uses. Do we really need to store all of this information?

Reply:
Other than the 2 columns you already commented, only `updatedAt` is being unused. But we might need that to calculate the time difference from last attempt on it. So yes, we need almost all of them. If you are worried about any column or its usage, please feel free to add comments so I can add details on how we are using or planning to use it.

Reply:
Luckily, we can make as many changes to the DB schema without doing migrations until we ship this code to customers. Which will be a while. So changes to the table is quick and easy for a while. 

Therefore, Let's keep the schema as-is for now. We can delete columns in a future refactor (before shipping to customers) if we feel we no longer need it. 

You may be right in that we need all of these columns but until I see code using the columns, I don't have much else to ask at this time.

Reply:
Yes, exactly. We can make as much changes we want before shipping the first version. I'll add this to action items so we can relook it once rest of the changes for BQ are merged, resolving it for now.